conda.recipe/meta.yaml

```diff
@@ -39,13 +39,14 @@ requirements:
     - clang 9
     - clangxx 9
     - rsync
-    # add runtime requirements for argos to make it easier to script 
+    # add runtime requirements for argos to make it easier to script
     # creation of a 'dev' environment that can also run Argos
     - matplotlib-base
     - numpy
     - pillow
     - pyyaml
     - wxpython
+    - boost
```

Comment:
Because of the way my janky environment creation scripts work right now, we need to duplicate `boost` in the run requirements and `ipython` in the build requirements.

Reply:
Done.  
1. Copied `ipython` to the `build` requirements
1. Copied `boost` to the `run` requirements